### PR TITLE
Add overlay short description on hypospray and pill bottles

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Title

image
image

## Why It's Good For The Game

Cm has it and I won't allow TGMC to fall behind in QOL in any department

## Changelog
:cl:
add: Add overlay short description on hypospray and pill bottles
/:cl:

### CODE REVIEW

**Comment by Arctrooper8** [comment]
Pictures? I'm confused by what this actually is.

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -193,6 +195,15 @@
 	icon_state = "hypo_d"
 
 
+/obj/item/reagent_containers/hypospray/update_overlays()
+	. = ..()
+	if(isturf(loc) || !description_overlay)
+		return
+	var/mutable_appearance/desc= mutable_appearance()
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/hypospray.dm:202]:
```suggestion
	var/mutable_appearance/desc = mutable_appearance()
```

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -411,11 +431,13 @@
 		/datum/reagent/medicine/meralyne = 30,
 		/datum/reagent/medicine/dermaline = 30,
 	)
+	description_overlay = "Md"
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/hypospray.dm:434]: Suggestion, maybe capitalize the D? Ie MD is 2 chems Md is one chem

File: code/game/objects/items/storage/firstaid.dm [hunk 3]
```diff
@@ -310,6 +312,12 @@
 	var/mutable_appearance/number = mutable_appearance()
 	number.maptext = MAPTEXT(length(contents))
 	. += number
+	if(!description_overlay)
+		return
+	var/mutable_appearance/desc= mutable_appearance()
```
Comment (TiviPlus) [inline code/game/objects/items/storage/firstaid.dm:317]:
```suggestion
	var/mutable_appearance/desc = mutable_appearance()
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]
As a side note could later make this setable by players

**Pull request merged**